### ISSUE TITLE
[WCAG] Project Filter: Low Contrast and Missing Fieldset Flagged

### ISSUE THREAD

**Issue by arielkidwell** [vigetlabs/pointless-corp-astro#17]
**Details**
![Image](https://github.com/user-attachments/assets/aa30fef9-ad7c-4f70-a2c2-d2dfe04293f8)
![Image](https://github.com/user-attachments/assets/0e16fd01-9220-4fa7-96d8-f76f8ad379e5)

**Comment by ten1seven** [comment]
@arielkidwell I updated the text color and wrapped the controls in fieldsets.
